### ISSUE TITLE
No move when every move draws by 50 moves repetition

### ISSUE THREAD

**Issue by rwbc** [znxftw/rudim#35]
As announced in the older memory issue, there is another (very rare) illegal move issue now.
Admittedly it only happens when draw adjudication is off, or very relaxed. (which I do exactly for finding otherwise hidden bugs)
Here is an example, 99 plies for the 50 moves rule draw were already played, but Rudim refused to play the 100th ply move,
when every move would have drawn immediately.

All other illegal moves in that 500 game match are about the same issue.

```
[Event "Rudim Test"]
[Site "RWBC-ESPRESSO Win11Pro Ryzen7 5700G 3.8Ghz 32GB"]
[Date "2024.12.09"]
[Round "78.1.1"]
[White "Rudim_13-2cf59c4-64"]
[Black "Stash_100-64"]
[Result "0-1"]
[ECO "A48"]
[GameDuration "00:02:57"]
[GameEndTime "2024-12-09T02:00:15.553 Mitteleuropäische Zeit"]
[GameStartTime "2024-12-09T01:57:17.669 Mitteleuropäische Zeit"]
[Opening "King's Indian"]
[PlyCount "268"]
[Termination "illegal move"]
[TimeControl "30+0.5"]
[Variation "London system"]

1. d4 {book} Nf6 {book}
2. Nf3 {book} g6 {book}
3. Bf4 {book} d6 {book}
4. Nbd2 {+0.50/6 0.76} Nh5 {-0.40/8 1.09}
5. Bg5 {+0.65/6 0.77} f6 {-0.13/7 1.08}
6. Be3 {+0.60/6 0.75} e5 {-0.06/7 1.07}
7. d5 {+0.45/6 0.75} Be7 {+0.19/7 1.06}
8. Bh6 {+0.75/6 0.75} c6 {+0.07/8 1.05}
9. e4 {+0.60/6 0.74} Nd7 {-0.06/7 1.03}
10. dxc6 {+1.07/7 0.75} bxc6 {+0.14/8 1.02}
11. Bc4 {+0.87/6 1.03} Bb7 {+0.11/7 1.01}
12. Nb3 {+0.60/6 1.02} a5 {+0.11/7 1.00}
13. O-O {+0.77/6 1.01} a4 {+0.27/7 0.99}
14. Nbd2 {+0.60/6 1.01} d5 {+0.28/7 0.98}
15. Bd3 {+0.72/7 0.98} Nc5 {+0.46/7 0.98}
16. Rb1 {+0.50/6 0.98} Nf4 {+0.70/7 0.96}
17. Bxf4 {+0.45/7 0.97} exf4 {+0.60/7 0.96}
18. Qe2 {+0.40/7 0.96} Qd7 {+0.62/7 0.94}
19. Rfe1 {+0.42/6 0.95} Nxd3 {+0.67/7 0.94}
20. Qxd3 {+0.45/6 0.94} O-O {+0.65/7 0.93}
21. Rbd1 {+0.45/6 1.05} Rae8 {+0.85/6 0.92}
22. Qd4 {+0.35/6 1.04} c5 {+0.80/7 0.91}
23. Qd3 {+0.30/6 1.03} Qe6 {+0.89/7 0.91}
24. exd5 {+0.40/7 1.00} Qxd5 {+0.87/7 0.90}
25. Qa3 {+0.40/7 1.01} Bc6 {+1.12/6 0.89}
26. h3 {+0.15/6 0.99} Kg7 {+1.12/7 0.88}
27. Kh2 {+0.20/6 0.97} Rf7 {+1.47/6 0.87}
28. Qc3 {+0.30/6 0.96} Rd8 {+1.30/7 0.88}
29. Qc4 {+0.33/5 0.94} Qxc4 {+1.43/7 0.86}
30. Nxc4 {+0.30/7 0.93} h6 {+1.41/8 0.86}
31. Rxd8 {+0.40/7 0.92} Bxd8 {+1.41/8 0.84}
32. Nfd2 {+0.33/7 0.92} Re7 {+1.30/7 0.83}
33. Rd1 {+0.25/7 0.90} Bd5 {+1.36/7 0.84}
34. Nd6 {+0.25/7 0.89} Bxa2 {+1.55/7 0.82}
35. N2e4 {+0.20/7 0.89} Re5 {+1.10/7 0.81}
36. Nc3 {+0.45/6 0.88} Bg8 {+1.35/8 0.81}
37. Nxa4 {+0.35/7 0.88} Be7 {+1.33/7 0.81}
38. Nc8 {+0.35/7 0.86} Bf8 {+1.05/8 0.80}
39. Rd7+ {+0.38/7 0.85} Bf7 {+1.25/8 0.79}
40. Nc3 {+0.38/7 0.84} g5 {+1.05/8 0.80}
41. Nb5 {+0.32/6 0.75} Kg8 {+1.28/8 0.79}
42. Rd2 {+0.20/7 0.75} Rd5 {+1.40/8 0.77}
43. Rxd5 {+0.14/8 0.76} Bxd5 {+1.20/9 0.77}
44. Ncd6 {0.00/8 0.75} Kh7 {+1.20/8 0.76}
45. g3 {-0.06/8 0.73} Kg6 {+1.50/9 0.76}
46. g4 {-0.09/9 0.73} Bc6 {+1.55/9 0.75}
47. Kg1 {-0.26/8 0.74} h5 {+1.70/9 0.74}
48. gxh5+ {-0.15/9 0.72} Kxh5 {+2.00/9 0.74}
49. Kh2 {-0.45/8 0.71} Bd7 {+2.15/9 0.74}
50. Kg2 {-0.50/8 0.72} Kh4 {+2.55/9 0.74}
51. b3 {-0.70/8 0.53} Be7 {+2.65/9 0.73}
52. c4 {-0.96/8 0.52} Bxh3+ {+3.20/10 0.72}
53. Kf3 {-0.89/9 0.52} f5 {+3.00/9 0.72}
54. Ke2 {-0.94/9 0.53} Kg4 {+2.95/9 0.71}
55. Nf7 {-1.04/8 0.51} Bf6 {+2.80/9 0.71}
56. Nh6+ {-1.07/8 0.52} Kh5 {+3.25/10 0.71}
57. Nf7 {-1.24/8 0.52} g4 {+3.10/9 0.71}
58. Nfd6 {-1.27/8 0.52} Kg6 {+3.35/8 0.69}
59. Nb7 {-1.22/7 0.52} Be7 {+3.05/9 0.69}
60. Nc3 {-1.23/8 0.52} Bg2 {+2.60/9 0.69}
61. Nd5 {-1.15/8 0.52} Bf3+ {+2.40/10 0.68}
62. Kd2 {-1.14/10 0.52} Bxd5 {+2.50/10 0.69}
63. cxd5 {-0.65/9 0.52} g3 {+2.20/9 0.68}
64. fxg3 {-0.46/9 0.51} fxg3 {+2.05/9 0.67}
65. Ke2 {-1.08/8 0.52} Bh4 {+1.45/9 0.67}
66. Kf3 {-0.58/9 0.52} Kf6 {+1.70/10 0.67}
67. Nxc5 {-0.60/9 0.52} Ke5 {+1.00/10 0.66}
68. Nd3+ {-0.52/9 0.52} Kxd5 {+1.75/10 0.66}
69. Nf4+ {-0.45/9 0.52} Kd4 {+0.75/10 0.66}
70. Ne2+ {-0.50/9 0.53} Ke5 {+0.70/10 0.65}
71. b4 {-0.43/10 0.52} g2 {+1.05/9 0.65}
72. b5 {-0.42/9 0.51} Bd8 {+1.15/9 0.66}
73. Kxg2 {-0.47/10 0.52} Ke4 {+1.10/9 0.65}
74. Ng3+ {-0.44/9 0.52} Kf4 {+1.10/9 0.64}
75. Nh5+ {-0.48/9 0.52} Kg4 {+1.50/10 0.64}
76. Ng3 {-0.71/10 0.51} f4 {+1.60/10 0.63}
77. Ne4 {-0.88/10 0.51} f3+ {+1.60/10 0.63}
78. Kf1 {-0.64/10 0.52} Kf5 {+1.30/9 0.64}
79. Nd6+ {-0.58/10 0.52} Kf4 {+1.50/10 0.63}
80. Nc4 {-0.23/9 0.51} Ke4 {-0.20/10 0.63}
81. b6 {+0.10/10 0.51} Kd5 {+0.30/10 0.62}
82. b7 {+0.02/10 0.52} Bc7 {+0.30/10 0.63}
83. Ne3+ {+0.17/9 0.52} Kc6 {+0.05/9 0.62}
84. Kf2 {+0.08/9 0.52} Kxb7 {+0.15/9 0.62}
85. Kxf3 {+0.08/9 0.52} Kc6 {+0.35/8 0.61}
86. Ke4 {+0.08/9 0.52} Kc5 {+0.45/8 0.61}
87. Nd5 {+0.08/10 0.51} Ba5 {+0.20/9 0.61}
88. Ne7 {+0.08/9 0.51} Bb4 {+0.50/8 0.61}
89. Ng8 {+0.08/9 0.52} Bd2 {+0.45/8 0.60}
90. Ne7 {+0.08/10 0.52} Bb4 {+0.50/8 0.60}
91. Ng8 {+0.08/9 0.52} Bd2 {+0.45/8 0.60}
92. Nf6 {+0.08/10 0.52} Be1 {+0.45/8 0.61}
93. Ne8 {+0.08/9 0.52} Bc3 {+0.30/9 0.60}
94. Nc7 {+0.08/10 0.52} Bf6 {+0.60/8 0.60}
95. Ne8 {+0.08/9 0.52} Bb2 {+0.30/9 0.59}
96. Nc7 {+0.03/10 0.52} Bf6 {+0.30/9 0.59}
97. Ne6+ {+0.08/9 0.51} Kd6 {+0.45/8 0.59}
98. Nd4 {+0.08/10 0.52} Kc5 {+0.50/8 0.59}
99. Ne6+ {+0.08/10 0.52} Kc6 {+0.40/8 0.59}
100. Nd4+ {+0.08/9 0.51} Kd6 {+0.40/8 0.58}
101. Nb5+ {+0.08/10 0.51} Kc5 {+0.25/9 0.58}
102. Nc7 {+0.08/10 0.51} Bb2 {+0.60/8 0.58}
103. Na6+ {+0.08/10 0.51} Kc4 {+0.25/9 0.58}
104. Nc7 {+0.08/10 0.52} Kc5 {+0.60/8 0.58}
105. Ne6+ {+0.08/9 0.52} Kd6 {+0.45/8 0.57}
106. Ng5 {+0.08/10 0.51} Ba1 {+0.50/8 0.57}
107. Nf7+ {+0.08/10 0.51} Kc5 {+0.45/8 0.57}
108. Nd8 {+0.08/10 0.51} Kc4 {+0.50/8 0.57}
109. Nf7 {+0.08/10 0.52} Kc5 {+0.45/8 0.57}
110. Nh6 {+0.08/9 0.51} Bc3 {+0.25/7 0.57}
111. Nf5 {+0.08/9 0.52} Bf6 {+0.40/8 0.57}
112. Nh6 {+0.08/10 0.52} Bc3 {+0.25/7 0.57}
113. Nf5 {+0.08/9 0.52} Bf6 {+0.40/8 0.56}
114. Ne3 {+0.08/9 0.52} Bg7 {+0.20/7 0.56}
115. Nd5 {+0.08/10 0.52} Kd6 {+0.45/8 0.56}
116. Nb4 {+0.08/9 0.51} Bf8 {+0.40/8 0.56}
117. Nd5 {+0.08/10 0.51} Bg7 {+0.20/9 0.56}
118. Nb4 {+0.08/9 0.52} Bf8 {+0.40/8 0.56}
119. Nd5 {+0.08/10 0.51} Bh6 {+0.40/8 0.56}
120. Nb6 {+0.08/10 0.51} Bg7 {+0.30/9 0.56}
121. Nc8+ {+0.08/10 0.51} Kc5 {+0.25/9 0.56}
122. Ne7 {+0.08/10 0.52} Bb2 {+0.50/8 0.56}
123. Ng8 {+0.08/9 0.52} Kd6 {+0.50/8 0.55}
124. Nh6 {+0.08/10 0.52} Kc6 {+0.50/8 0.55}
125. Ng8 {+0.08/9 0.51} Kd7 {+0.50/8 0.57}
126. Nh6 {+0.08/10 0.52} Ke6 {+0.25/7 0.56}
127. Nf5 {+0.08/10 0.51} Ba1 {+0.35/8 0.55}
128. Nh6 {+0.08/10 0.52} Kd6 {+0.50/8 0.56}
129. Nf5+ {+0.08/9 0.52} Kc5 {+0.40/8 0.54}
130. Nh4 {+0.08/10 0.51} Bf6 {+0.50/8 0.54}
131. Nf3 {+0.04/10 0.51} Bc3 {0.00/12 0.54}
132. Ne5 {+0.08/10 0.52} Kd6 {0.00/10 0.54}
133. Nf3 {+0.08/10 0.52} Bf6 {0.00/13 0.55}
134. Nd4 {+0.08/10 0.52} Be5 {0.00/13 0.54}
{White makes an illegal move: NoSquareNoSquare}
0-1
```

**Comment by znxftw** [comment]
Should be fixed with d284768b00935c2222cf0e3697685711461a1c00 - will add some tests for this and update here

**Comment by rwbc** [comment]
It is fixed, in a new test match I saw several games, where it now played the 100th ply move for drawing.
Ofc also no other illegal moves anymore. You can close this issue.

An example:
```
[Event "Rudim_Test"]
[Site "RWBC-ESPRESSO Win11Pro Ryzen7 5700G 3.8Ghz 32GB"]
[Date "2024.12.12"]
[Round "159.1.2"]
[White "Rudim_13-43dcdc7-64"]
[Black "Stash_100-64"]
[Result "1/2-1/2"]
[ECO "D06"]
[GameDuration "00:04:03"]
[GameEndTime "2024-12-12T03:50:32.568 Mitteleuropäische Zeit"]
[GameStartTime "2024-12-12T03:46:28.784 Mitteleuropäische Zeit"]
[Opening "Queen's Gambit declined"]
[PlyCount "395"]
[TimeControl "30+0.5"]
[Variation "Symmetrical (Austrian) defense"]

1. d4 {book} d5 {book}
2. c4 {book} c5 {book}
3. cxd5 {book} Qxd5 {book}
4. Nf3 {+0.50/6 0.77} cxd4 {-0.38/7 1.09}
5. Nc3 {+0.30/6 0.76} Qd7 {-0.32/7 1.08}
6. Qxd4 {+0.50/7 0.75} Nf6 {-0.35/8 1.07}
7. e4 {+0.30/6 0.75} Qxd4 {-0.20/8 1.06}
8. Nxd4 {+0.33/7 0.75} e5 {-0.33/8 1.04}
9. Ndb5 {+0.24/6 0.75} Na6 {-0.40/7 1.03}
10. Bg5 {+0.30/6 0.75} Bb4 {-0.30/7 1.02}
11. O-O-O {+0.44/7 1.02} O-O {-0.53/7 1.01}
12. Bxf6 {+0.45/7 1.01} gxf6 {+0.05/8 1.00}
13. Nd5 {+0.49/7 1.01} Kg7 {-0.10/8 0.99}
14. Nxa7 {+0.60/7 0.99} Rxa7 {+0.12/8 0.98}
15. Bxa6 {+0.50/7 0.99} Bc5 {+0.15/8 0.97}
16. Bc4 {+0.50/7 0.98} Ra4 {+0.12/7 0.96}
17. Bd3 {+0.30/6 0.97} Rxa2 {+0.12/7 0.96}
18. Kb1 {+0.25/6 0.96} Ra4 {+0.30/7 0.95}
19. f4 {+0.29/6 0.95} Be6 {+0.55/7 0.94}
20. Rc1 {+0.29/7 0.95} Bxd5 {+1.40/7 0.93}
21. exd5 {+0.19/7 1.05} Be3 {+1.70/8 0.92}
22. fxe5 {-0.58/7 1.04} Bxc1 {+2.75/8 0.92}
23. exf6+ {-0.68/8 1.03} Kxf6 {+2.80/8 0.90}
24. Rxc1 {-0.63/7 1.01} Ke5 {+2.65/7 0.89}
25. Bc4 {-0.65/7 1.00} Rb4 {+3.00/7 0.89}
26. Rc2 {-0.77/7 0.99} b5 {+3.10/7 0.88}
27. Re2+ {-0.72/8 0.97} Kd6 {+3.25/8 0.87}
28. Ba2 {-0.72/8 0.97} Ra8 {+3.05/7 0.86}
29. h3 {-0.60/7 0.96} h5 {+3.00/7 0.86}
30. Rc2 {-0.56/7 0.94} h4 {+3.10/7 0.86}
31. Rc6+ {-0.53/7 0.94} Ke5 {+3.15/8 0.84}
32. d6 {-0.66/8 0.92} Rf4 {+2.85/8 0.84}
33. d7 {-0.90/8 0.92} Rd8 {+2.85/7 0.84}
34. Rc5+ {-0.78/7 0.89} Kd6 {+2.85/7 0.82}
35. Rxb5 {-0.90/8 0.89} Rxd7 {+2.35/7 0.81}
36. Rd5+ {-0.70/7 0.88} Kc6 {+2.65/10 0.81}
37. Rxd7 {-0.78/8 0.88} Kxd7 {+2.45/9 0.81}
38. Bd5 {-1.01/8 0.86} Rf2 {+2.70/8 0.80}
39. Ka2 {-1.01/8 0.85} Kd6 {+2.70/8 0.79}
40. Bf3 {-0.99/9 0.83} Ke5 {+2.60/8 0.78}
41. Kb3 {-0.99/9 0.76} Kd4 {+2.40/9 0.78}
42. Bc6 {-0.99/9 0.75} Re2 {+2.70/8 0.77}
43. Bf3 {-1.01/8 0.74} Rd2 {+2.40/9 0.77}
44. Bc6 {-0.99/9 0.73} Re2 {+2.70/8 0.76}
45. Bf3 {-1.01/8 0.74} Rf2 {+2.70/8 0.76}
46. Bc6 {-0.99/9 0.73} Rd2 {+2.60/8 0.75}
47. Bf3 {-0.99/9 0.73} Kd3 {+2.40/9 0.75}
48. Bh5 {-0.99/9 0.72} f6 {+2.40/9 0.74}
49. Bf3 {-0.99/9 0.71} Rf2 {+2.40/9 0.74}
50. Bd5 {-0.99/9 0.72} Rd2 {+2.40/9 0.73}
51. Ka3 {-0.99/9 0.52} Kd4 {+2.65/9 0.73}
52. Bf3 {-1.08/8 0.52} Kc4 {+2.80/9 0.72}
53. Be4 {-1.10/8 0.52} Rd1 {+3.05/8 0.72}
54. Ka2 {-1.03/9 0.52} Rd7 {+3.05/8 0.71}
55. Kb1 {-1.03/8 0.52} Rd4 {+3.15/8 0.71}
56. Bb7 {-1.03/8 0.52} Rd7 {+3.10/8 0.71}
57. Ba8 {-1.03/8 0.51} Kb3 {+3.15/8 0.70}
58. Kc1 {-1.03/8 0.52} Rc7+ {+3.00/9 0.70}
59. Kb1 {-0.88/9 0.53} Rd7 {+2.85/9 0.69}
60. Kc1 {-1.03/8 0.52} Rd4 {+2.80/9 0.69}
61. Bf3 {-0.76/8 0.52} Rc4+ {+2.75/9 0.69}
62. Kb1 {-0.71/9 0.51} Rd4 {+2.80/9 0.68}
63. Kc1 {-0.70/9 0.52} Rc4+ {+2.75/9 0.69}
64. Kb1 {-0.71/9 0.52} Kb4 {+2.75/9 0.67}
65. Be2 {-0.82/8 0.52} Rc7 {+2.95/8 0.67}
66. Bg4 {-0.75/8 0.51} Rc4 {+2.95/8 0.68}
67. Be2 {-0.82/8 0.52} Rc8 {+2.95/8 0.67}
68. Bg4 {-0.74/8 0.52} Rc6 {+2.90/8 0.66}
69. Bf3 {-0.74/8 0.52} Rc8 {+2.75/9 0.66}
70. Bg4 {-0.74/8 0.51} Rc6 {+2.90/8 0.65}
71. Bf3 {-0.74/8 0.51} Rc7 {+2.80/8 0.65}
72. Bg4 {-0.75/8 0.51} Rc5 {+2.80/8 0.65}
73. Bd7 {-0.94/8 0.52} Rd5 {+2.95/8 0.65}
74. Bc6 {-0.94/8 0.52} Rd2 {+3.00/8 0.64}
75. Kc1 {-1.04/8 0.52} Rf2 {+2.85/9 0.64}
76. Ba8 {-1.10/8 0.52} Rf1+ {+2.85/9 0.64}
77. Kc2 {-1.10/10 0.52} Rf2+ {+2.85/9 0.63}
78. Kb1 {-1.03/9 0.53} Kb3 {+2.85/9 0.63}
79. Bd5+ {-1.03/9 0.52} Ka4 {+2.85/9 0.63}
80. Ka2 {-1.04/8 0.52} Kb4 {+2.90/9 0.63}
81. Bf3 {-1.03/9 0.52} Rf1 {+3.05/9 0.63}
82. Ba8 {-1.03/9 0.51} Rc1 {+3.05/9 0.63}
83. Be4 {-0.99/8 0.52} Re1 {+3.05/8 0.62}
84. Bd3 {-0.94/8 0.51} Re5 {+3.10/8 0.62}
85. Kb1 {-0.91/8 0.51} Kb3 {+3.20/8 0.61}
86. Bc2+ {-0.83/9 0.52} Kc4 {+3.10/8 0.61}
87. Bd1 {-0.93/8 0.52} Kd3 {+2.95/8 0.61}
88. Ka2 {-0.99/8 0.51} Re1 {+3.00/8 0.61}
89. Bh5 {-0.94/8 0.52} Kc4 {+3.00/8 0.60}
90. Bf7+ {-0.89/9 0.52} Kc5 {+3.05/8 0.60}
91. Bh5 {-0.94/8 0.52} Kb4 {+3.05/8 0.60}
92. Bg4 {-0.95/8 0.51} Rg1 {+3.10/8 0.60}
93. Bf3 {-1.01/8 0.51} Re1 {+3.05/9 0.60}
94. Bg4 {-0.95/8 0.51} Rg1 {+3.10/8 0.60}
95. Bf3 {-0.95/9 0.51} f5 {+3.00/10 0.59}
96. Ba8 {-1.03/9 0.52} Re1 {+2.95/9 0.59}
97. Bb7 {-1.04/8 0.52} Rc1 {+2.90/9 0.59}
98. Ba8 {-1.04/8 0.52} Re1 {+2.95/9 0.59}
99. Bb7 {-1.04/8 0.51} Rc1 {+2.90/9 0.59}
100. Ba8 {-1.04/8 0.52} f4 {+2.95/9 0.58}
101. Bb7 {-1.10/8 0.52} Rg1 {+2.95/9 0.58}
102. Ba8 {-1.03/9 0.52} Rc1 {+2.95/9 0.58}
103. Bb7 {-1.10/8 0.52} Rg1 {+2.95/9 0.58}
104. Ba8 {-1.03/9 0.52} Rh1 {+2.95/9 0.58}
105. Bb7 {-1.10/8 0.52} Rc1 {+2.95/9 0.57}
106. Ba8 {-1.10/8 0.51} Rh1 {+2.95/9 0.58}
107. Bb7 {-1.10/8 0.52} Rc1 {+2.95/9 0.57}
108. Ba8 {-1.10/8 0.52} Rd1 {+2.95/9 0.58}
109. Bb7 {-1.10/8 0.52} Rh1 {+2.95/8 0.57}
110. Ba8 {-1.10/8 0.52} Rd1 {+2.95/9 0.57}
111. Bb7 {-1.10/8 0.52} Rh1 {+2.95/8 0.57}
112. Ba8 {-1.10/8 0.52} Kc4 {+2.90/9 0.57}
113. Bf3 {-1.05/8 0.52} Rg1 {+2.90/9 0.57}
114. Be2+ {-1.03/9 0.52} Kb4 {+2.90/9 0.56}
115. Bf3 {-1.10/8 0.52} Re1 {+2.90/8 0.56}
116. Ba8 {-1.10/8 0.51} Re6 {+2.90/8 0.56}
117. Bf3 {-1.04/8 0.52} Re1 {+2.90/8 0.56}
118. Ba8 {-1.10/8 0.51} Kc4 {+2.90/8 0.56}
119. Bf3 {-1.05/8 0.51} Re7 {+2.95/8 0.56}
120. Kb1 {-1.04/8 0.51} Kd3 {+2.80/8 0.56}
121. Bg4 {-1.05/8 0.51} Re5 {+3.10/8 0.55}
122. Bf3 {-1.04/8 0.52} Rb5 {+3.05/8 0.55}
123. Bd1 {-1.04/8 0.51} Rg5 {+3.05/8 0.55}
124. Bf3 {-1.04/8 0.51} Rb5 {+3.05/8 0.55}
125. Bd1 {-1.04/8 0.52} Rg5 {+3.05/8 0.55}
126. Bf3 {-1.04/8 0.52} Rc5 {+2.85/8 0.55}
127. Bb7 {-1.03/8 0.52} Ra5 {+3.00/8 0.55}
128. Bc6 {-1.03/8 0.52} Rc5 {+2.95/8 0.55}
129. Bb7 {-1.03/8 0.52} Ra5 {+3.00/8 0.55}
130. Bc6 {-1.03/8 0.52} Ra7 {+2.95/8 0.54}
131. Bb5+ {-1.02/8 0.51} Ke3 {+2.85/8 0.54}
132. Bc6 {-1.00/8 0.51} Ra6 {+2.95/8 0.54}
133. Bf3 {-1.00/8 0.51} Kd3 {+2.85/8 0.54}
134. Bg4 {-1.03/8 0.52} Ra5 {+3.05/8 0.55}
135. Bc8 {-1.04/8 0.52} Rg5 {+2.95/8 0.54}
136. Bb7 {-1.05/8 0.52} Rg7 {+2.95/8 0.55}
137. Bf3 {-1.04/8 0.51} Rd7 {+2.85/8 0.54}
138. Ka2 {-1.05/8 0.52} Kc4 {+2.90/8 0.54}
139. Be2+ {-1.10/8 0.51} Kb4 {+2.90/8 0.54}
140. Bf3 {-1.10/8 0.51} Rd2 {+2.90/8 0.54}
141. Ba8 {-1.10/8 0.51} Rc2 {+2.90/8 0.54}
142. Bb7 {-1.03/9 0.52} Rf2 {+2.90/8 0.53}
143. Ba8 {-1.03/9 0.51} Re2 {+2.90/8 0.54}
144. Bb7 {-1.10/8 0.52} Rf2 {+2.90/8 0.53}
145. Ba8 {-1.03/9 0.51} Rd2 {+2.90/8 0.53}
146. Bb7 {-1.10/8 0.51} Rd4 {+2.90/8 0.53}
147. Ba8 {0.00/9 0.50} f3 {+0.95/9 0.53}
148. Bxf3 {+0.07/9 0.52} Rd2 {+1.00/9 0.53}
149. Ba8 {+0.07/9 0.51} Rd1 {+1.00/9 0.53}
150. Bb7 {0.00/8 0.51} Re1 {+1.00/8 0.53}
151. Ba8 {0.00/8 0.51} Rc1 {+1.00/9 0.53}
152. Bb7 {0.00/8 0.51} Rg1 {+1.00/9 0.53}
153. Ba8 {+0.07/9 0.51} Rc1 {+1.00/9 0.53}
154. Bb7 {0.00/8 0.52} Rg1 {+1.00/9 0.53}
155. Ba8 {+0.07/9 0.52} Rh1 {+1.00/9 0.53}
156. Bb7 {+0.07/9 0.51} Rd1 {+1.00/9 0.53}
157. Ba8 {0.00/8 0.51} Rh1 {+1.00/9 0.53}
158. Bb7 {+0.07/9 0.51} Rd1 {+1.00/9 0.52}
159. Ba8 {0.00/8 0.51} Re1 {+0.95/8 0.53}
160. Bb7 {0.00/8 0.52} Rc1 {+1.00/9 0.52}
161. Ba8 {+0.07/9 0.51} Rd1 {+1.00/9 0.53}
162. Bb7 {0.00/8 0.51} Rc1 {+1.00/9 0.52}
163. Ba8 {+0.07/9 0.52} Re1 {+0.95/8 0.52}
164. Bb7 {0.00/8 0.51} Rh1 {+1.00/8 0.52}
165. Ba8 {+0.07/9 0.52} Rg1 {+0.85/9 0.52}
166. Bb7 {+0.07/9 0.51} Rh1 {+1.00/8 0.52}
167. Ba8 {+0.07/9 0.52} Rg1 {+0.85/9 0.52}
168. Bb7 {+0.07/9 0.51} Re1 {+0.95/8 0.52}
169. Ba8 {0.00/8 0.51} Kc4 {+0.95/8 0.53}
170. Bf3 {+0.05/8 0.51} Re7 {+1.00/8 0.52}
171. Kb1 {+0.06/8 0.51} Kb4 {+0.90/8 0.52}
172. Kc2 {+0.13/8 0.52} Rc7+ {+0.90/8 0.52}
173. Kb1 {+0.27/9 0.52} Kc4 {+0.90/8 0.52}
174. Kc2 {+0.22/9 0.52} Kb4+ {+0.90/8 0.52}
175. Kb1 {+0.27/9 0.52} Rc8 {+0.85/8 0.52}
176. Bb7 {+0.16/8 0.51} Rc4 {+0.90/8 0.52}
177. Bf3 {+0.09/8 0.50} Rc7 {+0.90/8 0.52}
178. Bg4 {+0.16/8 0.51} Rc4 {+0.90/8 0.52}
179. Be2 {+0.16/8 0.52} Rc8 {+0.90/8 0.52}
180. Bg4 {+0.16/8 0.51} Rc4 {+0.90/8 0.52}
181. Be2 {+0.16/8 0.51} Rc8 {+0.90/8 0.51}
182. Bg4 {+0.16/8 0.52} Rc6 {+0.75/8 0.52}
183. Bh5 {+0.16/8 0.51} Kb3 {+0.75/8 0.52}
184. Bf7+ {+0.32/9 0.51} Kb4 {+0.55/9 0.52}
185. Bh5 {+0.16/8 0.51} Re6 {+0.65/8 0.51}
186. Bg4 {+0.06/8 0.51} Re1+ {+1.00/8 0.51}
187. Ka2 {+0.07/9 0.50} Kc4 {+0.95/8 0.51}
188. Bh5 {+0.05/8 0.51} Re5 {+1.05/8 0.51}
189. Bg4 {+0.05/8 0.51} Ra5+ {+1.10/8 0.51}
190. Kb1 {+0.10/9 0.51} Kd3 {+1.10/8 0.51}
191. Bc8 {+0.06/8 0.51} Ke3 {+1.10/8 0.51}
192. Bb7 {+0.08/8 0.51} Kd4 {+1.10/8 0.51}
193. Bc6 {+0.08/8 0.51} Ra6 {+1.10/8 0.51}
194. Bf3 {0.00/9 0.50} Kd3 {0.00/8 0.51}
195. Bd1 {0.00/9 0.52} Rf6 {0.00/10 0.51}
196. Bh5 {0.00/8 0.51} Rf1+ {0.00/10 0.51}
197. Ka2 {0.00/12 0.51} Ke4 {0.00/12 0.51}
198. Be8 {0.00/9 0.50}
{Draw by fifty moves rule}
1/2-1/2
```
